clients/cli/Cargo.toml

```diff
@@ -28,6 +27,7 @@ solana-signature = { workspace = true }
 solana-signer = { workspace = true }
 solana-system-interface = { workspace = true }
 solana-transaction = { workspace = true }
+spl-associated-token-account = { workspace = true }
```

Comment:
nit: can be down in follow-up work but `spl-associated-token-account-client` https://crates.io/crates/spl-associated-token-account-client/2.0.0 will give you what you need with fewer deps

(also, doesn't matter since the dep will change, but if using `spl-associated-token-account`, the `no-entrypoint` feature should also be enabled)

Reply:
Simple enough to update that dep 👍